Src/GoogleApis.Auth.DotNet4/OAuth2/ServiceAccountCredential.cs

```diff
@@ -142,8 +161,7 @@ public ServiceAccountCredential(Initializer initializer) : base(initializer)
             // Sign the header and the payload.
             var hashAlg = new SHA256CryptoServiceProvider();
             byte[] assertionHash = hashAlg.ComputeHash(Encoding.ASCII.GetBytes(assertion.ToString()));
-
-            var signature = UrlSafeBase64Encode(key.SignHash(assertionHash, "2.16.840.1.101.3.4.2.1" /* SHA256 OIG */)); 
+            var signature = UrlSafeBase64Encode(key.SignHash(assertionHash, "2.16.840.1.101.3.4.2.1" /* SHA256 OIG */));

```

Comment:
Hmm... this is weird. I don't see any difference between the contents of the two lines. I do see a blank line that got deleted. I will add that back.